metadata-models/src/main/pegasus/com/linkedin/platformresource/PlatformResourceInfo.pdl

```diff
@@ -0,0 +1,57 @@
+namespace com.linkedin.platformresource
+
+import com.linkedin.common.SerializedValue
+
+/**
+ * Platform Resource Info.
+ * These entities are for miscelaneous data that is used in non-core parts of the system.
+ * For instance, if we want to persist & retrieve data from auxiliary integrations such as Slack or Microsoft Teams.
+ */
+@Aspect = {
+  "name": "platformResourceInfo"
+}
+record PlatformResourceInfo {
+   /**
+    * The type of the resource. 
+    * Intended as a loose specifier of the generic type of the resource.
+    * Producer is not forced to conform to a specific set of symbols for
+    * resource types.
+    * The @PlatformResourceType enumeration offers a paved path for agreed upon
+    * common terms, but is not required to be followed.
+    * Example values could be: conversation, user, grant, etc.
+    * Resource types are indexed for ease of access. 
+    * e.g. Get me all platform resources of type user for the platform looker
+   */
+   @Searchable = {
```

Comment:
While these are not enabled for general search at the moment, if they ever are we probably want to consider which of these searchable fields should be queried by default from the main search bar. Ones that should not be included should be marked with `"queryByDefault": false`. If this entity will never be involved in main search then no need.

Reply:
The plan is to NOT make these resources visible in the Search UI ... are you recommending I add queryByDefault: false to make sure this never happens accidentally?

Reply:
No need. If this entity is ever added to entity types to search by default in the graphql layer, I've created a test to ensure this is considered.